models/data/dow_companies.sql

```diff
@@ -86,6 +86,54 @@ UNION
 select * from xom
 )comp;
 
-select * from companies
+select * from companys where company ="Amex" and date = "8/3/2018";
```

Comment:
Shouldn't this be companies as well.  But probably doesn't matter since this was just test code